deafrica/tests/test_latency_checker.py

```diff
@@ -0,0 +1,9 @@
+from deafrica.monitoring.latency_check import latency_checker
+import pytest
+
+
+def test_latency_checker():

```

Comment:
it is hard to write unit test for datacube as it require database setup. As far as I am aware we have used docker-compose to setup datacube test database and application/script and then executed a tests in other projects. 

For eg. https://github.com/opendatacube/datacube-docker/blob/main/statistician/test_statistician.sh